web/src/api/hooks/models.ts

```diff
@@ -49,6 +50,68 @@ export function addModel(data: Model): Promise<Model> {
     })(JSON.stringify(data));
 }
 
+export function deployModel(id: string): Promise<{ [key: string]: string }> {
+    const body = {
+        id
+    };
+    return useBadgerFetch<{ [key: string]: string }>({
+        url: `${namespace}/models/deploy`,
+        method: 'post',
+        withCredentials: true
+    })(JSON.stringify(body));
+}
+
+export function editModel(data: Model): Promise<Model> {
+    const body = { ...data };
+    // TODO: replace this hardcode with real data (update UI)
```

Comment:
I think it would be better to have task or detailed explanation here as nobody will remember what exactly need to be changed in future

Reply:
add issue https://github.com/epam/badgerdoc/issues/571